ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/PaywallDataExtensions.kt

```diff
@@ -0,0 +1,51 @@
+package com.revenuecat.purchases.ui.revenuecatui
+
+import android.util.Log
+import androidx.compose.foundation.isSystemInDarkTheme
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.ReadOnlyComposable
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.platform.LocalConfiguration
+import androidx.core.os.ConfigurationCompat
+import androidx.core.os.LocaleListCompat
+import com.revenuecat.purchases.paywalls.PaywallData
+import java.util.Locale
+
+@Composable
+@ReadOnlyComposable
+internal fun PaywallData.getColors(): PaywallData.Configuration.Colors {
+    return config.colors.dark?.takeIf { isSystemInDarkTheme() } ?: config.colors.light
```

Comment:
`isSystemInDarkTheme` a static function? Is it possible to override that for snapshot tests for example?

Reply:
Well, it's a composable function provided by Compose. I believe it can be overriden by hardcoding the light/night mode for the context, but for snapshot testing, we can configure the theme we're using so we can do snapshot tests for dark themes.